addons/controllers/webhooks/manifests.yaml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: mutating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    url: "https://127.0.0.1:9443/mutate-cni-tanzu-vmware-com-v1alpha1-antreaconfig"
```

Comment:
is there any reason to use a service instead here? see https://github.com/kubernetes/api/blob/b8c40e080bc5e830097df540d4ef804034cb5bdb/admissionregistration/v1/types.go#L498
 
hardconding the ip seems could bring down some problems say if a firewall or some sort of access is on (like selinux). using a service would allow for the actual cluster itself to correctly setup the service (host:port part of the url).